sklearn/metrics/cluster/tests/test_supervised.py

```diff
@@ -55,6 +66,20 @@ def test_perfect_matches():
         assert_equal(score_func([0., 1., 0.], [42., 7., 42.]), 1.0)
         assert_equal(score_func([0., 1., 2.], [42., 7., 2.]), 1.0)
         assert_equal(score_func([0, 1, 2], [42, 7, 2]), 1.0)
+    score_funcs_with_changing_means = [
+        normalized_mutual_info_score,
+        adjusted_mutual_info_score,
+    ]
+    means = {"min", "geometric", "arithmetic", "max"}
```

Comment:
feel kinda weird about calling these means.

Reply:
I can switch to `generalized_means` if you prefer, but it has to be clear that this is a specific class of aggregations. Product, for instance, wouldn't work.